RocksDBStore/RocksDBStore.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>3.0.0-CI00810</Version>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RootNamespace>Neo.Plugins.Storage</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Update="RocksDBStore\config.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Neo" Version="3.0.0-CI00810" />
+    <PackageReference Include="RocksDbNative" Version="6.2.2" />
```

Comment:
The test is failing because this error, but the nuget is there :S
```
   Test method neo_plugins.Tests.StoreTest.TestRocksDb threw exception: 
System.TypeInitializationException: The type initializer for 'RocksDbSharp.Native' threw an exception. ---> NativeImport.NativeLoadException: Unable to locate rocksdb native library, either install it, or use RocksDbNative nuget package
```

Reply:
Does it support `ubuntu-latest`?

Reply:
https://github.com/warrenfalk/rocksdb-sharp/issues/43#issuecomment-439638621